app/routes/task_routes.py

```diff
@@ -1 +1,77 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response, Response
+from app.models.task import Task
+from datetime import datetime
+import os
+import requests
+from ..db import db
+from .helper_methods import validate_model, helper_model_from_dict, helper_get_sorted_query
+
+bp = Blueprint("tasks_bp", __name__, url_prefix = "/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+
+    return helper_model_from_dict(Task, request_body)
+    
+@bp.get("")
+def get_all_tasks():
+    sort_param = request.args.get("sort") 
+    query = helper_get_sorted_query(Task, sort_param)
+    tasks = db.session.execute(query).scalars()
+    task_list = [task.to_dict() for task in tasks]
+    return task_list
```

Comment:
Could also directly return the list since you create `task_list` and then don't do anything with it except return it:
```suggestion
    return [task.to_dict() for task in tasks]
```